src/XrdSsi/XrdSsiResponder.cc

```diff
@@ -207,7 +207,7 @@ void XrdSsiResponder::ReleaseRequestBuffer()
   
 XrdSsiResponder::Status XrdSsiResponder::SetMetadata(const char *buff, int blen)
 {
-   XrdSsiMutexMon(spMutex);
+   XrdSsiMutexMon lck(spMutex);
 
 // If we don't have a request or the args are invalid, return an error.
 //
```

Comment:
You may get merge conflicts here if you didn't use the latest githead.

Reply:
I did rebase against the latest git HEAD :-) so no conflicts :-)